src/ProjectNeon/Assets/Data/Templates/DumbAI.cs

```diff
@@ -14,7 +14,7 @@ public override PlayedCard Play(Enemy activeEnemy)
     {
         var me = activeEnemy.AsMember();
         var card = activeEnemy.Deck.Cards.Random();
-        var possibleTargets = battleState.GetPossibleEnemyTeamTargets(me, card.TargetGroup, card.TargetScope);
+        var possibleTargets = battleState.GetPossibleEnemyTeamTargets(me, card.Actions[0].Group, card.Actions[0].Scope);
```

Comment:
We need Targets for each Effect in Played card, not just the first.

Reply:
@SilasReinagel My bad, I was thinking in a dumb ai just chosing the first target; I've fixed it